### PR TITLE
fix envoy examples

### PR DESCRIPTION
**Pull request by constambeys**
For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Commit Message: fix envoy examples
Additional Description:
Risk Level: Low
Testing: Manual Testing
Docs Changes: N/A
Release Notes:

### CODE REVIEW

**Comment by zuercher** [comment]
Could you provide some context around the problem this is fixing?

**Comment by constambeys** [comment]
Hi I am new to envoy and I was trying to run the examples and found these 2 errors.
The first error is that the image envoyproxy/envoy:latest does not exist in docker and the other error is that the container was not starting up because the entrypoint was pointing directly to the script.
I fixed this errors I think should be like this

File: examples/front-proxy/Dockerfile-service [hunk 1]
```diff
@@ -6,4 +6,4 @@ RUN mkdir /code
 ADD ./service.py /code
 ADD ./start_service.sh /usr/local/bin/start_service.sh
 RUN chmod u+x /usr/local/bin/start_service.sh
-ENTRYPOINT /usr/local/bin/start_service.sh
+ENTRYPOINT /bin/bash /usr/local/bin/start_service.sh
```
Comment (lizan) [inline examples/front-proxy/Dockerfile-service:9]: You should change the shebang to bash in the script instead.

Comment (constambeys) [reply examples/front-proxy/Dockerfile-service]: I didn't see the shebang, I changed the entrypoint to /bin/sh

Comment (lizan) [reply examples/front-proxy/Dockerfile-service]: https://github.com/envoyproxy/envoy/blob/master/examples/front-proxy/start_service.sh#L1 This line?

Comment (constambeys) [reply examples/front-proxy/Dockerfile-service]: yes

Comment (lizan) [reply examples/front-proxy/Dockerfile-service]: So why did you need this change?

Comment (constambeys) [reply examples/front-proxy/Dockerfile-service]: I get the following error:
docker logs -f a342aa41445bb0d63298f1bd695d1ab12cf9bec99b5ce35cc79c2fbdb98c15f6
/bin/sh: /usr/local/bin/start_service.sh: not found

**Pull request closed without merging**